src/Subsystems/Climber.cpp

```diff
@@ -18,10 +18,11 @@ std::shared_ptr<Climber> Climber::getInstance() {
   return self;
 }
 
-Climber::Climber() : Subsystem("Climber"),
+Climber::Climber() : Subsystem("kSusystemName"),
```

Comment:
Umm, let's try this again. Should be

    Climber::Climber() : Subsystem(kSubsystemName),